scripts/communities.js

```diff
@@ -126,7 +126,7 @@ async function getCommunityMinMaxDetails(communityName) {
   };
 
   function getValueForField(field, model) {
-    const fieldValues = model[field].split(' - ');
+    const fieldValues = model[field].split('-');
     if (fieldValues.length === 2) {
```

Comment:
What if there is a space in the spreadsheet before or after the hyphen?
Shouldn't the fieldValues be trimmed for spaces after splitting?

Reply:
parseFloat takes care of the space. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/parseFloat